code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -319,6 +319,33 @@ GLOBAL_VAR_INIT(hhmysteryRoomNumber, 1337)
     var/datum/turf_reservation/reservation
     var/turf/storageTurf
 
+/area/hilbertshotel/Entered(atom/movable/AM)
+    . = ..()
+    if(istype(AM, /obj/item/hilbertshotel))
+        relocate(AM)
+    var/list/obj/item/hilbertshotel/hotels = AM.GetAllContents(/obj/item/hilbertshotel)
+    for(var/obj/item/hilbertshotel/H in hotels)
+        if(parentSphere == H)
+            relocate(H)
+ 
+/area/hilbertshotel/proc/relocate(obj/item/hilbertshotel/H)
+    if(prob((1/737)*100)) //Because screw you
```

Comment:
now what have you done?

Reply:
cursed line

Reply:
it's `0.13568521031`

is prob(0.13) too fancy for you?

Reply:
Because screw you

Reply:
@optimumtact Ugh, a 13/1000 chance...😡 But 1/737 chance... 😍